xml/hwbp_registers.xml

```diff
@@ -245,7 +250,12 @@
             When set, every instruction completed or exception taken in M mode decrements \Fcount
             by 1.
         </field>
-        <field name="0" bits="8" access="R" reset="0" />
+        <field name="hit" bits="8" access="R/W" reset="0">
```

Comment:
same comment as above related to the placement of this bit.

Reply:
I moved the hit bits.